config/base_boxes.yaml

```diff
@@ -76,3 +76,13 @@ boxes:
     box: centos7
     <<: *install_shell
     options: --install-type=devel
+
+  centos6-foreman:
+    box: centos6
+    <<: *install_shell
+    options: --install-type=foreman
+
+  centos7-foreman:
+    box: centos7
+    <<: *install_shell
+    options: --install-type=foreman
```

Comment:
Can you add boxes for Foreman versions? For example, I wanted to spin up a Foreman 1.9 box this morning to test Remote Execution.